flight/apps/telemetry/packing.py

```diff
@@ -0,0 +1,344 @@
+"""
+
+Telemetry packing for transmission
+
+Each fixed-length LoRa Payload is structured as follows
+MESSAGE_ID : 1 byte
+SEQ_COUNT  : 2 bytes
+PACKET_LENGTH: 1 byte
+PACKET_DATA  : 252 bytes (for now)
+
+For struct module format strings:
+    "b": 1,  # byte
+    "B": 1,  # unsigned byte
+    "h": 2,  # short
+    "H": 2,  # unsigned short
+    "i": 4,  # int
+    "I": 4,  # unsigned int
+    "l": 4,  # long
+    "L": 4,  # unsigned long
+    "q": 8,  # long long
+    "Q": 8,  # unsigned long long
+    "f": 4,  # float
+    "d": 8,  # double
+
+"""
+from apps.telemetry.constants import ADCS_IDX, CDH_IDX, EPS_IDX, GPS_IDX, THERMAL_IDX
+from apps.telemetry.helpers import (
+    convert_float_to_fixed_point_hp,
+    pack_signed_long_int,
+    pack_signed_short_int,
+    pack_unsigned_long_int,
+    pack_unsigned_short_int,
+    unpack_unsigned_short_int,
+)
+from core import logger
+from core.data_handler import DataHandler as DH
+from micropython import const
+
+
+class TelemetryPacker:
```

Comment:
Kind of no point in making this a class if every single method has an `@classmethod` decorator

Reply:
No instantiation, the class acts as a namespace for the shared state
Avoid global scope pollution and maintain consistency with the rest of the codebase